frontend/src/visualization/shared-ops/convertQueryResult.ts

```diff
@@ -11,7 +11,7 @@ function convertNode(node: NodeResultDescriptor): vis.Node {
   const result: vis.Node = {
     id: node.id,
     label: node.id.toString(),
-    color: colorize(node.types).color,
+    color: colorize({ id: node.id, types: node.types }).color,
```

Comment:
we wanted to make use of the store. ATM the store is useless since the Graph will never be updated when the store changes.
Change function to `function convertNode(node: NodeResultDescriptor, colorize: [...]): vis.Node`

and modify Graph to listen to changes in the coloring functions.
```ts
const colorizeNodes = useObservable(nodeColorStore.getState());
const colorizeEdges = useObservable(nodeColorStore.getState());
const queryResult = useObservable(queryResultStore.getState());

const graphData = convertQueryResult(queryResult, colorizeNodes, colorizeEdges);
```